packages/react-vapor/src/components/limit/Limit.tsx

```diff
@@ -19,98 +20,69 @@ export interface LimitOwnProps {
+type HeaderProps = Pick<LimitProps, 'limitLabel' | 'isHistoryIncluded' | 'onClick'>;
+type HistoryIconProps = Pick<HeaderProps, 'isHistoryIncluded' | 'onClick'>;
 type ProgressBarProps = Pick<LimitProps, 'usage' | 'isLimitTheGoalToReach' | 'limit'>;
 
-const LimitDisconnect: React.FunctionComponent<LimitProps> = ({
+export const Limit: React.FunctionComponent<LimitProps> = ({
     id,
     title,
     limitLabel = 'Limit',
     usage,
     limit = 100,
-    currentLimit,
     isLimitEditable = false,
     isLimitTheGoalToReach = false,
     isHistoryIncluded = false,
-    onChangeLimit,
     className,
-}) => (
-    <div className={classNames('limit-box mb2', className)}>
-        <div className="limit-box-main p2 pb1">
-            <HeaderDivision limitLabel={title} isHistoryIncluded={isHistoryIncluded} />
-            <ContentDivision
-                id={id}
-                usage={usage}
-                limit={limit}
-                isLimitEditable={isLimitEditable}
-                limitLabel={limitLabel}
-                onChangeLimit={onChangeLimit}
-            />
+    onClick,
+}) => {
+    const {currentLimit} = useSelector((state: IReactVaporState) => ({
+        currentLimit: +InputSelectors.getValue(state, {id}) || (limit ?? 100),
+    }));
+
+    return (
+        <div className={classNames('limit-box mb2', className)}>
+            <div className="limit-box-main p2 pb1">
+                <HeaderDivision limitLabel={title} isHistoryIncluded={isHistoryIncluded} onClick={onClick} />
+                <ContentDivision
+                    id={id}
+                    usage={usage}
+                    limit={limit}
+                    isLimitEditable={isLimitEditable}
+                    limitLabel={limitLabel}
+                />
+            </div>
+            <ProgressBar usage={usage} isLimitTheGoalToReach={isLimitTheGoalToReach} limit={currentLimit} />
         </div>
-        <ProgressBar usage={usage} isLimitTheGoalToReach={isLimitTheGoalToReach} limit={currentLimit} />
-    </div>
-);
+    );
+};
 
-const HeaderDivision: React.FunctionComponent<HeaderProps> = ({limitLabel, isHistoryIncluded}) => (
+const HeaderDivision: React.FunctionComponent<HeaderProps> = ({limitLabel, isHistoryIncluded, onClick}) => (
     <div className="flex space-between">
         <label className="form-control-label"> {limitLabel}</label>
-        <HistoryIcon isHistoryIncluded={isHistoryIncluded} />
+        <HistoryIcon isHistoryIncluded={isHistoryIncluded} onClick={onClick} />
     </div>
 );
 
-const HistoryIcon: React.FunctionComponent<HistoryIconProps> = ({isHistoryIncluded}) =>
+const HistoryIcon: React.FunctionComponent<HistoryIconProps> = ({isHistoryIncluded, onClick}) =>
     isHistoryIncluded && (
-        <span className="icon mod-lg limit-history-button">
+        <span className="icon limit-history-button documentation-link" onClick={onClick}>
             <Svg svgName={VaporSVG.svg.menuAnalytics.name} />
         </span>
```

Comment:
Why not use a LinkSvg component here?

Reply:
I always forget about this component. Will switch